src/Fizzics/level1.js

```diff
@@ -27,36 +32,23 @@ var FizzicsLevel1 = GObject.registerClass({
 }, class FizzicsLevel1 extends Gtk.Grid {
     _init(props = {}) {
         super._init(props);
-        this._backgroundGroup = new ButtonGroup({
-            buttonBackground0: this._buttonBackground0,
-            buttonBackground1: this._buttonBackground1,
-            buttonBackground2: this._buttonBackground2,
-        });
-
-        this._editor0 = new FizzicsObjectEditor({visible: true});
-        this._image0 = new FizzicsSkinImage({visible: true, pixels: 32});
-        this._tab0.add(this._editor0);
-        this._tab0Image.add(this._image0);
-
-        this._editor1 = new FizzicsObjectEditor({visible: true});
-        this._image1 = new FizzicsSkinImage({visible: true, pixels: 32});
-        this._tab1.add(this._editor1);
-        this._tab1Image.add(this._image1);
 
-        this._editor2 = new FizzicsObjectEditor({visible: true});
-        this._image2 = new FizzicsSkinImage({visible: true, pixels: 32});
-        this._tab2.add(this._editor2);
-        this._tab2Image.add(this._image2);
-
-        this._editor3 = new FizzicsObjectEditor({visible: true});
-        this._image3 = new FizzicsSkinImage({visible: true, pixels: 32});
-        this._tab3.add(this._editor3);
-        this._tab3Image.add(this._image3);
+        const indices = Array.from({length: 3}, (v, i) => i);
+        this._menuBackground = new PopupMenu(this._buttonBackground, indices,
+            FizzicsSkinImage, 'index',
+            {path: '/com/endlessm/HackToolbox/Fizzics/backgrounds'});
```

Comment:
I am having a hard time understanding the purpose of this change in this commit. `PopupMenu` does not have a `path` property.

Reply:
This parameter would be a dict of extra properties given to the created FizzicsSkinImage

Reply:
Oh yeah! I was confused when looking at `PopupMenu` and thought that the property dict was going to the menu itself.